internal/contour/handler.go

```diff
@@ -80,14 +83,34 @@ type opDelete struct {
 }
 
 func (e *EventHandler) OnAdd(obj interface{}) {
+	obj, err := e.converter.ConvertUnstructured(obj)
```

Comment:
Is it possible to pull the e.Error change out into its own change, this looks like a worthwhile change in and of itself.

Reply:
Hey, not following what you are asking? The `e.Error` is new from the converter logic.